src/components/MessagesList/MessagesGroup/Message/MessageButtonsBar/MessageButtonsBar.vue

```diff
@@ -583,32 +596,24 @@ export default {
 				? moment().add(1, 'weeks').day(1).hour(8)
 				: null
 
-			return [
-				{
-					key: 'laterToday',
-					timestamp: this.getTimestamp(laterTodayTime),
-					label: t('spreed', 'Later today – {timeLocale}', { timeLocale: laterTodayTime?.format('LT') }),
-					ariaLabel: t('spreed', 'Set reminder for later today'),
-				},
-				{
-					key: 'tomorrow',
-					timestamp: this.getTimestamp(tomorrowTime),
-					label: t('spreed', 'Tomorrow – {timeLocale}', { timeLocale: tomorrowTime?.format('ddd LT') }),
-					ariaLabel: t('spreed', 'Set reminder for tomorrow'),
-				},
-				{
-					key: 'thisWeekend',
-					timestamp: this.getTimestamp(thisWeekendTime),
-					label: t('spreed', 'This weekend – {timeLocale}', { timeLocale: thisWeekendTime?.format('ddd LT') }),
-					ariaLabel: t('spreed', 'Set reminder for this weekend'),
-				},
-				{
-					key: 'nextWeek',
-					timestamp: this.getTimestamp(nextWeekTime),
-					label: t('spreed', 'Next week – {timeLocale}', { timeLocale: nextWeekTime?.format('ddd LT') }),
-					ariaLabel: t('spreed', 'Set reminder for next week'),
-				},
-			].filter(option => option.timestamp !== null)
+			const times = {
+				'Later today': laterTodayTime,
+				Tomorrow: tomorrowTime,
+				'This weekend': thisWeekendTime,
+				'Next week': nextWeekTime,
+			}
+
+			return Object.entries(times).map(([key, moment]) => {
+				return {
+					key: key.toLowerCase().replace(' ', '_'),
```

Comment:
this does not match the previous keys? Would it be important?

Reply:
It is only used as key in loop and not elsewhere